pages/portfolio.html

```diff
@@ -5,8 +5,31 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles.css" rel="stylesheet">
 </head>
 <body>
-  
+  <main class = "container">
+    <header>
+      <h1>Emily's Site</h1>
+      <nav class = "menu">
+        <a href = "index.html" id  = Home>Home</a>
+        <a href = "about.html" id = About>About Me</a> 
+        <a href = "portfolio.html" id = Portfolio>Portfolio</a>
+      </nav>
+    </header>
+    <section class = "middle">
+      <h2>My Projects</h2>
+      <nav>
```

Comment:
the nav tag is intended as a way to group together the main navigation section on a site, like you did with the menu links in the header. We don't need to wrap a nav tag around every anchor tag in the page.